src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -187,12 +188,19 @@ public int handleExecutionException(Exception ex, CommandLine cmd, ParseResult p
         printPointerToLogFile = false;
       } else {
         errorMessage =
-            "An unexpected error occurred in "
+            "An unexpected error occurred: "
                 + ex.getClass().getCanonicalName()
                 + ": "
-                + ex.getMessage();
+                + ex.getMessage()
+                + "\n";
         formattedErrorMessage =
-            systemAndUnexpectedErrorStyle.errorText("[ERROR] ").concat(errorMessage);
+            systemAndUnexpectedErrorStyle
```

Comment:
For a while, I was winding up with unexpected errors, but no stack trace printed, so I added this. I _think_ it's not strictly necessary, as I was getting duplicate stack traces after fixing the logging.